kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -359,6 +368,15 @@ public class FunSpec private constructor(
       typeVariables += typeVariable
     }
 
+    @ExperimentalKotlinPoetApi
+    public fun contextReceiver(receiverTypes: Collection<TypeName>): Builder = apply {
+      check(!name.isConstructor) { "$name: constructors cannot have context receivers" }
```

Comment:
Will name here ever be different from `constructor`? If no - should we omit it from the error message?

Reply:
I was asking myself the same question, but this is consistent with receiver and @ZacSweers suggested it

Reply:
Got it, sounds good!